ff --git a/deps/v8/src/base/logging.h b/deps/v8/src/base/logging.h
index 08db24a9..7fb824ed 100644
--- a/deps/v8/src/base/logging.h
+++ b/deps/v8/src/base/logging.h
@@ -8,7 +8,7 @@
 #include <cstring>
 #include <sstream>
 #include <string>
-
+#include <inttypes.h>
 #include "src/base/base-export.h"
 #include "src/base/build_config.h"
 #include "src/base/compiler-specific.h"
diff --git a/deps/v8/src/inspector/v8-string-conversions.h b/deps/v8/src/inspector/v8-string-conversions.h
index c1d69c18..9e78c344 100644
--- a/deps/v8/src/inspector/v8-string-conversions.h
+++ b/deps/v8/src/inspector/v8-string-conversions.h
@@ -5,6 +5,7 @@
 #ifndef V8_INSPECTOR_V8_STRING_CONVERSIONS_H_
 #define V8_INSPECTOR_V8_STRING_CONVERSIONS_H_

+#include <inttypes.h>
 #include <string>

 // Conversion routines between UT8 and UTF16, used by string-16.{h,cc}. You may
